model.py

```diff
@@ -3211,9 +3215,9 @@ def target_age_string(self):
         upper = self.target_age.upper
         if not upper and not lower:
             return ""
-        if lower and not upper:
+        if lower and upper is None:
             return str(lower)
-        if upper and not lower:
+        if upper and lower is None:
```

Comment:
We have 0-8 as the age range for early grades in the metadata spreadsheet for the basque titles, and it was showing up as "8".